src/main/java/app/dto/task/CreateTaskRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package app.dto.task;
+
+import app.model.Priority;
+import app.model.TaskStatus;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+import lombok.Data;
+import org.springframework.format.annotation.DateTimeFormat;
+
+@Data
+public class CreateTaskRequestDto {
+    @NotBlank
+    private String name;
```

Comment:
```suggestion
    @NotBlank(message = "Task name must be defined"
    private String name;
```
also check other fields